nemo/collections/nlp/modules/common/megatron/transformer.py

```diff
@@ -229,12 +228,6 @@ def __init__(
                 self.normalization = get_layer_norm(
                     ffn_hidden_size // get_tensor_model_parallel_world_size(), layernorm_epsilon, persist_layer_norm
                 )
-            elif normalization == 'layernorm1p':
```

Comment:
Why did you remove these? It is likely an issue with incorrect rebasing to main. Please restore these bits.